pkg/monitor/cluster/cluster.go

```diff
@@ -67,6 +67,10 @@ func NewMonitor(env env.Interface, log *logrus.Entry, oc *api.OpenShiftCluster,
 func (mon *Monitor) Monitor(ctx context.Context) error {
 	mon.log.Debug("monitoring")
 
+	mon.emitGauge("cluster.version", 1, map[string]string{
+		"version": mon.oc.Properties.ClusterProfile.Version,
```

Comment:
This reports the value we've got in the DB. Unfortunately it can become stale when user decide to upgrade their cluster by directly changing the cluster objects.

I think, we need to send a request to the API server to get the actual version.

I'm dealing with the similar issue, but in context of the RP API in #274. See `pkg/util/clusterdata/cluster_version_task.go`

Reply:
I've seen how it should have been done and updated the PR now.